tests/lists.test.ts

```diff
@@ -1,15 +1,47 @@
 import { PyInt, PyBool, PyNone, PyObj } from "../utils";
 import { assert, asserts, assertPrint } from "./utils.test";
 
-describe("run", () => {
-  // assert('add', "2 + 3", PyInt(2 + 3));
-  // assert('Lists Declaration','items : [int] = None\nitems', PyNone())
-  // var source = `
-  //     items : [int] = None
-  //     n : int = 10
-  //     items = [1, 2]
-  //     n = items[0] + items[1]
-  //     n
-  // `
-  // assert('Lists Access', source,  PyInt(1 + 2))
+describe("LIST TEST", () => {
+  var source = `
+        items : [int] = None
+        items = [1, 2, 3]
+    `;
+
+  assert("Assign List To Variable (Program 2)", source, PyNone());
+
+  var source = `
+    items : [int] = None
+    items = [1, 2, 3] + [4, 5, 6]
+    items[5]
+    `;
+
+  assert("Concat List (Program 3)", source, PyInt(6));
+
+  var source = `
+        items : [int] = None
+        n : int = 10
+        items = [1, 2]
+        n = items[0] + items[1]
+        n
+    `;
+
+  assert("Lists Access (Program 4)", source, PyInt(1 + 2));
+
+  //Other Test
+  assert("Empty List", "[]", PyObj(`list<none>`, 540));
+  assert("List With Number", "[1,2,3]", PyObj(`list<number>`, 632));
```

Comment:
I think these tests are brittle in the presence of any memory management/changing allocation strategies. Maybe it would be best to find a way to check just the list type part, and let the address be anything?